develop/core/config.py

```diff
@@ -1,15 +1,9 @@
-from django.contrib.sites.models import Site
 from django.utils.translation import gettext_lazy as _
 from siteconfigs.config import SiteConfigBaseClass
 from .forms import ExampleForm
 
 
 class ExampleClass(SiteConfigBaseClass):
-    label = _("Example")
-    default = {"example": "Default Value"}
-    form_class = ExampleForm
-
-    def __init__(self):
-        site = Site.objects.get_current()
-        self.key = ".".join([__name__, __class__.__name__])
-        super().__init__(site, self.key)
+    _label = _("Example")
+    _form_class = ExampleForm
+    example = "Default Value"
```

Comment:
wondering; is there some way of typing these attributes? ie

class ExampleClass(SiteConfigBaseClass):
    _label = _("Example")
    _form_class = ExampleForm
    example = "Default Value"
    is_available: bool = False
    
 That way we might be able to validate submitted json data? although I guess that's what the form class is for just wondering if we can steal some behavior from [pydantic](https://docs.pydantic.dev/latest/concepts/models/#validation)

Reply:
If we think types are a necessary attribute in the JSON, we can make the change for how the data is stored.
`{foo:{value: 1, type:int}}`
That will add some complexity so I would only do it if we really think it would be useful.
The other avenue is to use a getter/setter pattern in the Config class and let the JSON store it in the basic form.